src/common/prop.c

```diff
@@ -85,7 +83,6 @@ daos_prop_fini(daos_prop_t *prop)
 	}
 
 	D_FREE(prop->dpp_entries);
-	prop->dpp_entries = NULL;
```

Comment:
Is it safe to remove this? What if there is code that relies on this being set to NULL?
I.e. D_FREE free's this, but does not set to NULL, correct?

Reply:
D_FREE() does set things to NULL.

Reply:
Oh? Good to know